wolfssl/test.h

```diff
@@ -997,14 +997,16 @@ static INLINE unsigned int my_psk_server_cb(WOLFSSL* ssl, const char* identity,
 #if !defined(WOLFSSL_MDK_ARM) && !defined(WOLFSSL_KEIL_TCP_NET)
     #include <sys/time.h>
 
-    static INLINE double current_time(void)
+    static INLINE double current_time(int reset)
     {
         struct timeval tv;
         gettimeofday(&tv, 0);
-
+        (void)reset;
```

Comment:
Good solution. Portable. Cast to void if not needed, we have access to the variable if it is needed.